osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -23,6 +23,68 @@ public OsuDifficultyCalculator(Ruleset ruleset, WorkingBeatmap beatmap)
         {
         }
 
+        public OsuDifficultySectionRating  = new List<double>();
```

Comment:
You forgot to specify the type of this field. And why is it public?

Reply:
i thinked about it as output of function at some time